src/main/java/gui/RootPane.java

```diff
@@ -45,7 +45,7 @@ public void start(Stage primaryStage) throws Exception {
         // created. Also has some useful settings.
         //TODO: change this line of code.
         Scene scene = new Scene(topLevelPane);
-        scene.getStylesheets().add("testcss.css");
+        scene.getStylesheets().add("stylesheet.css");
         primaryStage.setScene(scene);
         primaryStage.setTitle("Hoi ben een titel lol.");
```

Comment:
Can't right now. CSS is applied to the entire scene for now, it doesn't seem to work properly when a stylesheet is applied to a single object only.

Furthermore, the stylesheet is applied recursively to every single object, and is accessible (using getStyleSheet) from every (every!) single object, already.